contrib/release/pull-docker-manifests.sh

```diff
@@ -69,19 +69,25 @@ main() {
     version="v${ersion}"
     run_url_id="$(basename "${3}")"
 
+    if [ ! -e "${PWD}/install/kubernetes/Makefile.digests" ]; then
+        >&2 echo "Cannot find install/kubernetes/Makefile.digests."
```

Comment:
nit: I would either omit the last dot or replace by a comma here, as-is it can be read as part of the path.